js/main.js

```diff
@@ -76,7 +76,7 @@ class DmgBlueTheme {
     prepare_users_list() {
         // Loop through the array of LightDMUser objects to create our user list.
         for (var user of lightdm.users) {
-            $(`<li id="${user.name}">${user.name}</li>`).appendTo(this.$users_list);
+            $(`<li id="${user.username}">${user.display_name}</li>`).appendTo(this.$users_list);
```

Comment:
Hmm I had put both as `user.username` , didn't know about `display_name` until now.

Example: `jsmith` vs. `John Smith`

I personally prefer the `username` as the text because it's like short and hacky looking, and what you are used to seeing in the terminal. Someone else might prefer `display_name` though. I guess it could easily be a setting as well.

Reply:
Yeah, personally I prefer to show display_name to be more user friendly, but it's a matter of taste~